ecl/hql/hqlutil.cpp

```diff
@@ -5525,11 +5525,30 @@ IHqlExpression * extractCppBodyAttrs(unsigned lenBuffer, const char * buffer)
 {
     OwnedHqlExpr attrs;
     unsigned prev = '\n';
+
     for (unsigned i=0; i < lenBuffer; i++)
     {
         char next = buffer[i];
         switch (next)
         {
+        case '*':
+            if ('/' == prev) // Ignore directives in multi-line comments
+            {
+                i+=2;
+                while (i < lenBuffer && ('*' == buffer[i-1] && '/' == buffer[i])) 
```

Comment:
Your right.  (I took the expression out from inside the loop, where it breaked if it matched.)